api-reference/10 UI Components/dxValidationSummary/3 Methods/refreshValidationGroup().md

```diff
@@ -3,7 +3,252 @@ id: dxValidationSummary.refreshValidationGroup()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Refreshes the ValidationSummary subscription to a target ValidationGroup.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Use this method if target ValidationGroup is recreated.
+
+---
+##### jQuery
+    <!-- tab: index.html -->
+    <div id="container"></div>
+    <div id="summary"></div>
+    <div id="validate-group"></div>
+    <div id="remove-group"></div>
+    <div id="render-group"></div>
+
+    <!-- tab: index.js -->
+    $(function() {
+
+        function renderGroup() {
+            $("#container").append(
+                $("<div></div>").dxTextBox({}) 
+                .dxValidator({
+                    validationRules: [{ type: 'required' }],
+                    validationGroup: "group"
+                }),
+                $("<div></div>").dxTextBox({}) 
+                .dxValidator({
+                    validationRules: [{ type: 'required' }],
+                    validationGroup: "group"
+                })
+            );
+        }
+
+        renderGroup();
+
+        const summary = $("#summary").dxValidationSummary({
+            validationGroup: "group"
+        }).dxValidationSummary('instance');
+
+        $("#validate-group").dxButton({
+            text: "Validate",
+            validationGroup: "group",
+            onClick: function(e) {
+                e.validationGroup.validate();
+            }
+        });
+
+        $("#remove-group").dxButton({
+            text: "Remove group",
+            onClick: function() {
+                $("#container").empty();
+            }
+        });
+
+        $("#render-group").dxButton({
+            text: "Render group",
+            onClick: function() {
+                renderGroup();
+                summary.refreshValidationGroup(); // Refresh the subscription to ValidationGroup after rendering
+            }
+        });
+  
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-validation-group #group>
+        <dx-text-box *ngIf="visible">
+            <dx-validator>
+                <dxi-validation-rule type="required">
+                </dxi-validation-rule>
+            </dx-validator>
+        </dx-text-box>
+        <dx-text-box *ngIf="visible">
+            <dx-validator>
+                <dxi-validation-rule type="required">
```

Comment:
I think it's more pretty to not separate closing tag if there is no nested component.

Smth like this:
`<dxi-validation-rule type="required" />`

Reply:
Noted